.github/scripts/docs-after-clone.sh

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
+set -e
+
+[[ -z "$1" ]] && echo "Missing working directory argument" && exit 1
+
+#echo "Installing root npm"
+#pnpm -C $1 i
+
+#echo "Running root build"
+#pnpm -C $1 --filter "*/meteor-icon-kit" build
+
+#echo "Building figma"
+#pnpm -C $1 --filter "*/meteor-icon-kit" start
```

Comment:
Can those be removed?

Reply:
I'll delete the whole file because the build step is not needed anymore and we can control some stuff from the `docs.yml`.